chain/default_syncer_test.go

```diff
@@ -952,7 +956,9 @@ func TestTipSetWeightDeep(t *testing.T) {
 	verifier = proofs.NewFakeVerifier(true, nil)
 	con = consensus.NewExpected(cst, bs, th.NewTestProcessor(), &consensus.MarketView{}, calcGenBlk.Cid(), verifier)
 	syncer := chain.NewDefaultSyncer(cst, con, chainStore, blockSource)
-	baseTS := chainStore.Head() // this is the last block of the bootstrapping chain creating miners
+	headTipSetAndState, err := chainStore.GetTipSetAndState(ctx, chainStore.GetHead())
```

Comment:
nit: a helper function `requireGetHeadTipSet` (or whichever better name you find) that runs `GetHead` and then `GetTipSetAndState` would help simplify tests for readers.